include/openspace/engine/openspaceengine.h

```diff
@@ -193,6 +199,9 @@ class OpenSpaceEngine {
     // Others
     std::unique_ptr<properties::PropertyOwner> _globalPropertyNamespace;
     
+    bool _switchScene;
```

Comment:
Bool variables should have some verb in them in order to make the if statements more readable and make them less ambigous For example `_shouldSwitchScene`  or `_scheduleSwitchScene`.